dev/com.ibm.ws.logging.core/src/com/ibm/ws/collector/manager/buffer/BufferManagerImpl.java

```diff
@@ -39,33 +39,31 @@
 	protected Queue<Object> earlyMessageQueue;
 
 	private static final int EARLY_MESSAGE_QUEUE_SIZE = 400;
-
+	private final static boolean defaultIsSoftRefEMQ = true;
```

Comment:
not sure this is really needed as a variable... suggest getting rid of defaultIsSoftRefEMQ and just update the ctor below to use hardcoded true value.